docs/book/src/faq.md

```diff
@@ -0,0 +1,88 @@
+
+# Kubebuilder FAQ
+
+## How to undestand the domain field?
+First of all, we should get familiar with the GVK terminology, check [here](./cronjob-tutorial/gvks.md).
```

Comment:
WDYT about something like:  

```suggestion
After creating a project, usually you will want to extend the Kubernetes APIs and define new APIs which will be owned by your project. Therefore, the domain value is tracked in the [PROJECT][project-file-def] file which defines the config of your project and will be used as a domain to create the endpoints of your API(s). Please, ensure that you understand the [Groups and Versions and Kinds, oh my!][gvk].
```

Note that for maintainability we keep the links at the bottom and we use an alias. You can check this standard applied in the other docs. 

So, let's add to the bottom:

``` 
[gkv]: ./cronjob-tutorial/gvks.md
[project-file-def]: https://book.kubebuilder.io/reference/project-config.html
```